qemu/qemu.go

```diff
@@ -1025,7 +1028,25 @@ func (config *Config) appendKnobs() {
 		config.qemuParams = append(config.qemuParams, "-daemonize")
 	}
 
-	if config.Knobs.MemPrealloc == true {
+	// Both HugePages and MemPrealloc require the memory size of the VM
```

Comment:
@mcastelino Sorry to be pedantic, but it would be better to have the comment in the Knobs structure as this will then appear in the godoc.

https://godoc.org/github.com/01org/ciao/qemu#Knobs

Reply:
@markdryan that should be addressed now. Showing up in godoc is required for the externally visible options.